controllers/search.controller.js

```diff
@@ -21,6 +22,22 @@ async function searchResults(req, res) {
     });
 }
 
+async function emailResults(req, res) {
+    let results = req.body.results;
+    let message;
+    if (results === undefined) {
```

Comment:
Error-handling should be done via `next(<err message>);`. See [this example for reference](https://github.com/hackmcgill/hackerAPI/blob/develop/middlewares/settings.middleware.js#L47).
This way, `*.controller.js` only handles successes.